infra/examples/aws-google-workspace/build

```diff
@@ -3,4 +3,14 @@
 # NOTE: Terraform modules *should* build deployment bundle themselves, but in case that fails, this
 # script will do the equivalent with its output directly in your terminal
 
-../../../tools/build.sh aws ../../../java
+
+TF_CONFIG_ROOT=`pwd`
+PSOXY_BASE_DIR=${TF_CONFIG_ROOT}/.terraform/modules/psoxy
```

Comment:
Uhm -not for this PR-, in such case -where the code comes from module, so in theory it should not be locally modified- I think best option could be to just download the JAR from GH too and not compiling it in local

Reply:
distributing the JAR has some complexity:
  - analyze how all the dependencies are licensed.  If GPL, we'd need to, at minimum, list those with links to the source of each. (if customer builds JAR, we aren't distributing dependences - we're just listing them in the pom.xml, and customer is using their own `Maven` to obtain them)
  - trust factor that JAR is what we say it is (compilation of JAR from minimal Java code which customers can review plus linking to a list dependencies obtained from whatever Maven repository your trust, is more trustworthy than "here, take this JAR")
  - extra steps in our release process to build + publish the artifacts for each release